.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,61 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "<title>"
+labels: [bug, performance, ci]
```

Comment:
Let's limit to `bug` and `triage` initially. Not all bugs would be CI or perf related (hopefully).